src/globals/zothacks-styles.scss

```diff
@@ -39,3 +39,15 @@ $white: #ffffff;
 $lightGray: #e8e8e8;
 $offWhite: #ededed;
 $pink: #ff8dcb;
+
+
+// Animations
+@keyframes wobble {
+    0% { transform: translateY(0%); } 
+    15% { transform: translateY(-1.5%) rotate(4deg); } 
+    30% { transform: translateY(1.0%) rotate(-3deg); } 
+    45% { transform: translateY(-1.0%) rotate(2deg); } 
+    60% { transform: translateY(0.5%) rotate(-2deg); }
+    75% { transform: translateY(-0.5%) rotate(1deg); }
+    100% { transform: translateY(0%); } 
+ } 
```

Comment:
Could you share how this looks with Philip and get some feedback?

Reply:
I have talked to Philip about this, and he said he was envisioning a more bouncing animation. The animation that is currently implemented is added [here](https://github.com/HackAtUCI/ZotHacks-Site/blob/feature/landing-jeep/src/app/views/home/sections/Landing.scss#L99) and placed the animations [here](https://github.com/HackAtUCI/ZotHacks-Site/blob/feature/landing-jeep/src/globals/zothacks-styles.scss#L43-L62). This animation has been run through and approved by Philip.

Reply:
Thanks for getting that feedback. I want to add that we might need to scale down the entire animation based on the size of the jeep to keep things proportional.

Reply:
While fixing the responsiveness issue on Safari, it seems to adjust the animation scale already without any additional changes to the animations themselves. I do want to confirm that the animation scale is sufficient before I continue making any more changes needed.

Reply:
The animation should be fine since it uses percentages, but I think we may need to adjust the position offset separately.